### DISCUSSION TITLE
How do we do testing for `Uuid::new_v4()`?

### DISCUSSION THREAD

**Discussion by jymchng** [uuid-rs/uuid#664] [General]
I am trying to do a unit test for replacing word that matches a regex with a `Uuid`, however, `Uuid::new_v4()` generates a random `Uuid` each time, how can I write such unit test?

**Comment by kinggoesgaming** [comment]
IMO you should check for the `version` and `variant` properties as they should be the same across any `v4` Uuid instance.
